### PR TITLE
Added the program for Node to Root Path in a Binary Tree

### PR DESCRIPTION
**Pull request by Jassi10000-zz**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
Yes

### Description

(Write your answer here.)
Successfully added the program for Node to Root Path in a Binary in Java folder 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#3706

### CODE REVIEW

File: Java/ds/NodetoRootPath.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        Node root = new Node(60);
+        root.left = new Node(50);
+        root.right = new Node(70);
+        root.left.left = new Node(30);
+        root.left.right = new Node(52);
+        root.right.left = new Node(65);
+        root.right.right = new Node(75);
+        root.right.left.left = new Node(62);
+        root.right.left.right = new Node(68);
+
```
Comment (HarshCasper) [inline Java/ds/NodetoRootPath.java]: Keep it dynamic

File: Java/ds/NodetoRootPath.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+}
+
+// O/P:
+// true
+// [62, 65, 70, 60]
+
+// Time Complexity : O(n) in Worst Case
+// Space Complexity : O(1)
```
Comment (HarshCasper) [inline Java/ds/NodetoRootPath.java]: Use proper words not lazy ones. Use multi-line comments

File: Java/ds/NodetoRootPath.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import java.util.ArrayList;
```
Comment (HarshCasper) [inline Java/ds/NodetoRootPath.java]: Add a beginner friendly pitch

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Java/README.md [hunk 4]
```diff
@@ -118,6 +118,7 @@ _add list here_
 
 - [Binary Tree](ds/Binary_Tree.java)
 - [Check if a Binary Tree is a Binary Search Tree](ds/CheckBST.java)
+- [Node to Root Path in a Binary Tree](https://github.com/Jassi10000/NeoAlgo/blob/298419ee98ccc42ef36e489a7af15776a49a2f85/Java/ds/NodetoRootPath.java)
```
Comment (ankitaggarwal23) [inline Java/README.md]: Kindly change it to like this `- [Node to Root Path in a Binary Tree](ds/NodetoRootPath.java)`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/ds/NodetoRootPath.java (hunk 3 above)
Comment (Jassi10000-zz) [reply Java/ds/NodetoRootPath.java]: Can you pls elaborate this thing , as I can't understand this

File: Java/ds/NodetoRootPath.java (hunk 1 above)
Comment (Jassi10000-zz) [reply Java/ds/NodetoRootPath.java]: Sure

File: Java/ds/NodetoRootPath.java (hunk 2 above)
Comment (Jassi10000-zz) [reply Java/ds/NodetoRootPath.java]: Sure , leme try to do that

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this PR due to inactivity.

**Comment by Jassi10000-zz** [comment]
@ankitaggarwal23  I was just making these changes only ,  
The reason for my inactivity was : I had fever , so was not able to do these things asked

I dont' know why all the project maintainers don't even ask the reason from the person who submitted the PR , and just close it 
It's really very rude of you guys

File: Java/ds/NodetoRootPath.java (hunk 3 above)
Comment (Jassi10000-zz) [reply Java/ds/NodetoRootPath.java]: @ankitaggarwal23  Tell me this thing , I asked this  doubt 2 days ago 
Tell me how many of you answered this

**Comment by ankitaggarwal23** [comment]
> @ankitaggarwal23 I was just making these changes only ,
> The reason for my inactivity was : I had fever , so was not able to do these things asked
> 
> I dont' know why all the project maintainers don't even ask the reason from the person who submitted the PR , and just close it
> It's really very rude of you guys

@Jassi10000 It's not like that as you're inactive from last 3 days after we requested the changes the thing is as the one month phase of gssoc is going to be an end and now we are finalizing.
You can start working.

**Comment by Jassi10000-zz** [comment]
> > @ankitaggarwal23 I was just making these changes only ,
> > The reason for my inactivity was : I had fever , so was not able to do these things asked
> > I dont' know why all the project maintainers don't even ask the reason from the person who submitted the PR , and just close it
> > It's really very rude of you guys
> 
> @Jassi10000 It's not like that as you're inactive from last 3 days after we requested the changes the thing is as the one month phase of gssoc is going to be an end and now we are finalizing.
> You can start working.

Thank you for understanding @ankitaggarwal23 for reopening this , I will deliver the requested changes as soon as possible

File: Java/ds/NodetoRootPath.java (hunk 3 above)
Comment (Jassi10000-zz) [reply Java/ds/NodetoRootPath.java]: @ankitaggarwal23 Sir , can you tell what Harsh means by Beginner Friendly Pitch  for this line

**Comment by Jassi10000-zz** [comment]
@ankitaggarwal23  Sir , can you please check nd review my latest commits , I have corrected all the faults that were present in my file and Readme too

**Review by Jassi10000-zz — COMMENTED** [review]
I have reviewed my code self , 
@ankitaggarwal23  Sir , you also have a look and give the review on my latest commits

File: Java/ds/NodetoRootPath.java [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+// I/P:
+// 19
+// 50 25 12 n n 37 30 n n n 75 62 n 70 n n 87 n n
+// 30
+// O/P:
+// true
+// [30, 37, 25, 50]
+
+// Worst Case Time Complexity : O(n)
+// Space Complexity : O(1)
```
Comment (HarshCasper) [inline Java/ds/NodetoRootPath.java]: Use Multi-Line Comments here

File: Java/ds/NodetoRootPath.java [hunk 6]
```diff
@@ -0,0 +1,150 @@
+import java.io.*;
```
Comment (HarshCasper) [inline Java/ds/NodetoRootPath.java:12]: Add beginner friendly description here

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Jassi10000-zz** [comment]
@HarshCasper I have made the requested changes , please check once

**Comment by HarshCasper** [comment]
Add a beginner friendly description first. Address the changes requested first

**Comment by Jassi10000-zz** [comment]
> Add a beginner friendly description first. Address the changes requested first

@HarshCasper Now check once

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by Jassi10000-zz** [comment]
@ankitaggarwal23 Thanks for merging the PR

**Pull request merged**